gradle/wrapper/gradle-wrapper.properties

```diff
@@ -2,6 +2,6 @@
 # SPDX-License-Identifier: Apache-2.0
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-7.4.2-bin.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-7.6.1-bin.zip
```

Comment:
Any reason not to bump gradle to 8.1.1 to match core?

Reply:
A line from gradle output
```
Deprecated Gradle features were used in this build, making it incompatible with Gradle 8.0.
```
I afraid something may not working. Just restarted build with `--warning-mode all`:
```
> Configure project :
The Report.enabled property has been deprecated. This is scheduled to be removed in Gradle 8.0. Please use the required property instead. See https://docs.gradle.org/7.6.1/dsl/org.gradle.api.reporting.Report.html#org.gradle.api.reporting.Report:enabled for more details.
        at build_2th86uuomtcgdejvuk5nrmh56$_run_closure7$_closure20.doCall(C:\GitHub\opensearch-project-sql-pr\build.gradle:132)
        (Run with --stacktrace to get the full stack trace of this deprecation warning.)

> Configure project :opensearch-sql-plugin
The AbstractArchiveTask.archiveName property has been deprecated. This is scheduled to be removed in Gradle 8.0. Please use the archiveFileName property instead. See https://docs.gradle.org/7.6.1/dsl/org.gradle.api.tasks.bundling.AbstractArchiveTask.html#org.gradle.api.tasks.bundling.AbstractArchiveTask:archiveName for more details.
        at build_z79fajveig7tp1hdihyed28n$_run_closure14$_closure30$_closure33.doCall(C:\GitHub\opensearch-project-sql-pr\plugin\build.gradle:227)
        (Run with --stacktrace to get the full stack trace of this deprecation warning.)
The AbstractArchiveTask.version property has been deprecated. This is scheduled to be removed in Gradle 8.0. Please use the archiveVersion property instead. See https://docs.gradle.org/7.6.1/dsl/org.gradle.api.tasks.bundling.AbstractArchiveTask.html#org.gradle.api.tasks.bundling.AbstractArchiveTask:version for more details.
        at build_z79fajveig7tp1hdihyed28n$_run_closure14$_closure30$_closure33.doCall(C:\GitHub\opensearch-project-sql-pr\plugin\build.gradle:228)
        (Run with --stacktrace to get the full stack trace of this deprecation warning.)
....
> Task :sql:generateGrammarSource
...
Gradle detected a problem with the following location: 'C:\GitHub\opensearch-project-sql-pr\sql\build\generated-src\antlr\main\org\opensearch\sql\sql\antlr\parser'. Reason: Task ':sql:generateEffectiveLombokConfig' uses this output of task ':sql:generateGrammarSource' without declaring an explicit or implicit dependency. This can lead to incorrect results being produced, depending on what order the tasks are executed. This behaviour has been deprecated and is scheduled to be removed in Gradle 8.0. Execution optimizations are disabled to ensure correctness. See https://docs.gradle.org/7.6.1/userguide/validation_problems.html#implicit_dependency for more details.
....
> Task :ppl:generateGrammarSource
...
Gradle detected a problem with the following location: 'C:\GitHub\opensearch-project-sql-pr\ppl\build\generated-src\antlr\main\org\opensearch\sql\ppl\antlr\parser'. Reason: Task ':ppl:generateEffectiveLombokConfig' uses this output of task ':ppl:generateGrammarSource' without declaring an explicit or implicit dependency. This can lead to incorrect results being produced, depending on what order the tasks are executed. This behaviour has been deprecated and is scheduled to be removed in Gradle 8.0. Execution optimizations are disabled to ensure correctness. See https://docs.gradle.org/7.6.1/userguide/validation_problems.html#implicit_dependency for more details.
....
> Task :legacy:generateGrammarSource
...
Gradle detected a problem with the following location: 'C:\GitHub\opensearch-project-sql-pr\legacy\build\generated-src\antlr\main\org\opensearch\sql\legacy\antlr\parser'. Reason: Task ':legacy:generateEffectiveLombokConfig' uses this output of task ':legacy:generateGrammarSource' without declaring an explicit or implicit dependency. This can lead to incorrect results being produced, depending on what order the tasks are executed. This behaviour has been deprecated and is scheduled to be removed in Gradle 8.0. Execution optimizations are disabled to ensure correctness. See https://docs.gradle.org/7.6.1/userguide/validation_problems.html#implicit_dependency for more details.
```